critique.md

```diff
@@ -1,273 +1,70 @@
-
-
-## path: ./IPNS/LRMECS/hdf5
-
-| file             | critique                              |
-| ---------------- | ------------------------------------- |
-| ``lrcs3701.nx5`` | NeXus HDF5 file, 2 **NXentry** groups |
-
-
-## path: ./Soleil/hdf5
-
-| file           | critique                             |
-| -------------- | ------------------------------------ |
-| ``file_1.nxs`` | NeXus HDF5 file, 1 **NXentry** group |
-| ``file_2.nxs`` | NeXus HDF5 file, 1 **NXentry** group |
-
-
-## path: ./SwissFEL
-
-| file       | critique      |
-| ---------- | ------------- |
-| ``README`` | not HDF5 file |
-
-
-## path: ./SwissFEL/hdf5
-
-| file                                   | critique                             |
-| -------------------------------------- | ------------------------------------ |
-| ``lyso009a_0087.JF07T32V01_master.h5`` | NeXus HDF5 file, 1 **NXentry** group |
-
-
-## path: ./code/hdf4
-
-| file           | critique      |
-| -------------- | ------------- |
-| ``NXtest.hdf`` | not HDF5 file |
-| ``dmc01.hdf``  | not HDF5 file |
-| ``dmc02.hdf``  | not HDF5 file |
-
-
-## path: ./code/hdf5
-
-| file                     | critique                              |
-| ------------------------ | ------------------------------------- |
-| ``NXtest.h5``            | NeXus HDF5 file, 2 **NXentry** groups |
-| ``dmc01.h5``             | NeXus HDF5 file, 1 **NXentry** group  |
-| ``dmc02.h5``             | NeXus HDF5 file, 1 **NXentry** group  |
-| ``focus2007n001335.hdf`` | NeXus HDF5 file, 1 **NXentry** group  |
-| ``sans2009n012333.hdf``  | NeXus HDF5 file, 1 **NXentry** group  |
-
-
-## path: ./code/xml
-
-| file               | critique      |
-| ------------------ | ------------- |
-| ``NXtest.xml.txt`` | not HDF5 file |
-| ``dmc01.xml.txt``  | not HDF5 file |
-| ``dmc02.xml.txt``  | not HDF5 file |
-
-
-## path: ./hdf5
-
-| file                        | critique                             |
-| --------------------------- | ------------------------------------ |
-| ``simple3D.h5``             | NeXus HDF5 file, 1 **NXentry** group |
-| ``writer_1_3.h5``           | NeXus HDF5 file, 1 **NXentry** group |
-| ``writer_1_3__niac2014.h5`` | NeXus HDF5 file, 1 **NXentry** group |
-
-
-## path: ./nxpdb
-
-| file            | critique      |
-| --------------- | ------------- |
-| ``4n8z.cif``    | not HDF5 file |
-| ``4n8z.h5.cif`` | not HDF5 file |
-| ``README.rst``  | not HDF5 file |
-
-
-## path: ./nxpdb/hdf5
-
-| file        | critique                             |
-| ----------- | ------------------------------------ |
-| ``4n8z.h5`` | NeXus HDF5 file, 1 **NXentry** group |
-
-
-## path: ./xml
-
-| file               | critique      |
-| ------------------ | ------------- |
-| ``verysimple.xml`` | not HDF5 file |
+* unimplemented test cases are marked in the table with an asterisk
+| path                                        | file                                     | File Type    | NXentry Count | Application Def's |
```

Comment:
add blank line before this to start a table

Reply:
OK, can do.